config/manager/manager.yaml

```diff
@@ -102,3 +102,26 @@ spec:
       - name: webhook-tls
         secret:
           secretName: webhook-server-tls
+---
+kind: PodDisruptionBudget
+metadata:
+  name: controller-manager-pdb
+  namespace: default
+spec:
+  minAvailable: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      app.kubernetes.io/name: k8s-pdb-autoscaler
+      app.kubernetes.io/managed-by: kustomize
```

Comment:
I'd remove this one as we're not even using kustomize at the moment and may rip it out in the future.